docs/paradigm-layouts.md

```diff
@@ -0,0 +1,171 @@
+exactly of the string `--` or a **wordform** cell.
+
+A wordform cell **may** contain a `${lemma}` placeholder. In which case,
+the paradigm layout is a dynamic layout, and the layout cells must be
+_filled_ before the layout is ready for presentation. Regardless if the
+wordform cell contains a `${lemma}` placeholder, there are few
+limitations in what can go in a wordform cell. As of now,
+a wordform cell is assumed to consist of either the literal wordform's
+text or an _FST analysis_ string.
+
+### Example
+
+Here is a paradigm layout file with two **panes**, each with
+a **header**, and each row with row labels and each column with column
+labels:
+
+Source: <https://docs.google.com/document/d/1nWQQvHgjGnXzOswm_TsnLx75Yp0-qzamTSm-7_INEsw/edit#>
+
+
+### Partial Grammar
+
+
+### Example
+
+
+
+[vd]: https://www.visidata.org/
+[pycharm-tsv]: https://www.jetbrains.com/help/pycharm/editing-csv-and-tsv-files.html
+
+
+Where to place paradigm layout files
+------------------------------------
+
+Paradigm layout files are placed in the following directory structure
+template:
+
+```
+layouts
+├── dynamic
+│   ├── {paradigm-name}
+│   │   ├── {size-1}.tsv
+│   │   ├── {size-2}.tsv
+│   │   └── {size-3}.tsv
+│   ├── {paradigm-name}
+│   └── {paradigm-name}
+└── static
+    ├── {paradigm-name}
+    │   ├── {size-1}.tsv
+    │   ├── {size-2}.tsv
+    │   └── {size-3}.tsv
+    ├── {paradigm-name}.tsv
+    └── {paradigm-name}.tsv
+```
+
+Layout files can either be _dynamic_ or _static_, and must be placed in
+the respective directory hierarchy. Files directly within the `dynamic`
+or `static` directories, can be:
+
+ * a `.tsv` layout file, whose filename stem (part before the `.tsv`) is
+   the _paradigm name_ that this layout corresponds to;
+ * a subdirectory, whose filename is the _paradigm name_ this directory
+   corresponds to. Within this subdirectory are `.tsv` layout files that
+   indicate the different _size options_ available for this _paradigm
+   name_.
+
+> **Note**: the exact location for the root `layouts` file depends on
+> the dictionary, however, this _should_ be with in the dictionary's
+> resources directory. As of 2021-06-10, this means
+> `src/CreeDictionary/res/layouts` for the `crkeng` dictionary.
+
+
+Glossary
+--------
+
+### Dynamic vs. Static
+
+A **dynamic paradigm layout** is one in which there are *placeholders*
+for the `${lemma}`. At runtime, the placeholders are filled with
+a provided lemma such that the placeholders can be substituted with
+concrete wordforms, to be presented to users.
+
+Dynamic paradigm layouts are useful for linguistic paradigms that are
+_productive_ (i.e., used for systematically for a large number of
+lexemes).
+
+A **static paradigm layout** is one in which there are no placeholders.
+All cells have their wordforms **explicitly** filled out. No
+substitution is needed at runtime, so their contents will be presented
+**as-is**.
+
+Static paradigm layouts are useful for
+
```

Comment:
Something missing here!

Reply:
These comments are somehow associated with an old commit; the current commit does not contain either of the lines commented upon. 🤔